palisade-service/src/main/java/uk/gov/gchq/palisade/service/palisade/response/ErrorResponse.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright 2020 Crown Copyright
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package uk.gov.gchq.palisade.service.palisade.response;
+
+import com.fasterxml.jackson.annotation.JsonAutoDetect;
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import org.springframework.util.Assert;
+
+/**
+ * Error message in human readable form.  This can be generated in any of the services.  Once an error occurs
+ * in a service,  processing of the request stops.  This message is constructed and forwarded to the results-service
+ * skipping any other services that have not been preformed.  The results-service will forward this message back
+ * to client who should be given enough information to correct the problem before tying again.
+ * The technical message will contain information that may help in understanding the issue and so may contain
+ * information about the service which should not be made public such as the stack trace of the error.
+ */
+@JsonAutoDetect(fieldVisibility = JsonAutoDetect.Visibility.ANY)
+public class ErrorResponse {
+
+
+    public final String technicalMessage; //Technical detail about where/when the error occurred.
+
+    public final String errorMessage;  //Detailed description of the error in english
```

Comment:
I feel the (in english) bit is a bit redundant, something pointing to this being a user-friendly discriptive output not meant for debugging purposes would be more appropriate